CefSharp.Core/Internals/Messaging/Messages.h

```diff
@@ -1,4 +1,4 @@
-// Copyright � 2010-2017 The CefSharp Authors. All rights reserved.
+// Copyright © 2010-2017 The CefSharp Authors. All rights reserved.
```

Comment:
Looks like you've changed the file encoding, unless absolutely necessary please revert.